.github/workflows/ci.yml

```diff
@@ -10,6 +10,22 @@ jobs:
   build:
```

Comment:
The name of the job here corresponds to the CI checks that are run.. I should have probably named it something better originally, but lets change this 🙂 

<img width="468" alt="Screen Shot 2021-11-18 at 17 00 52" src="https://user-images.githubusercontent.com/482871/142461707-65f7ca6d-bfeb-43c6-bd7c-e6b8467b670f.png">


```suggestion
  checks:
```